src/local-fs.ts

```diff
@@ -72,15 +72,19 @@ export class LocalFileSystem implements FileSystem {
                             type: 'fileCreated',
                             fullPath: relPath.split(path.sep).join(pathSeparator),
                             newContent: content
-                        })));
+                        }))
+                        .catch(() => {})
```

Comment:
silently swallowing errors is very unfriendly to the user. we need to avoid such errors completely. if we can't achieve that, reporting them in some way (that is not necessarily part of the `FileSystem` API) is the lesser evil